tests/drop.rs

```diff
@@ -50,7 +50,10 @@ async fn test_node_drop() -> Result<()> {
 
     tracing::info!("Dropped {} at {}", dropped_name, dropped_addr);
 
-    for (node, _) in &mut nodes {
+    //  A failed send_message from any node should
+    // trigger voting by all nodes
+    {
+        let (node, _) = nodes.get(0).ok_or_else(|| anyhow!("Missing Node"))?;
```

Comment:
better pick the `last` node, not the first one.
as the first one is normally a genesis, which may have some privilege.